### PR TITLE
Move security exclusion from .bandit.ini to the corresponding code line

### PR DESCRIPTION
**Pull request by francesco-giordano**
### Description of changes
* Move security exclusion from .bandit.ini to the corresponding code line. This change aims to avoid global security exclusions.
* The security exclusions added by this PR will be assessed as part of another PR.
* Added the label skip-security-exclusions-check since now the checks are more precise.
* Also added `profile=black` in isort.cfg to avoid clashing between isort and black8

Please review the [guidelines for contributing](../CONTRIBUTING.md) and [Pull Request Instructions](https://github.com/aws/aws-parallelcluster/wiki/Git-Pull-Request-Instructions).

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: cli/src/pcluster/api/util.py [hunk 1]
```diff
@@ -11,7 +11,10 @@
 import datetime
 import logging
 import shutil
-import subprocess
+
+# A nosec comment is appended to the following line in order to disable the B404 check.
+# In this file the input of the module subprocess is trusted.
+import subprocess  # nosec
```
Comment (awslitvit) [inline cli/src/pcluster/api/util.py]: Nitpick.
`# nosec B404` or even a more speaking `# nosec import_subprocess` should be safer, even if in this specific case it is pretty the same.

File: cli/src/pcluster/cli/commands/dcv_connect.py [hunk 2]
```diff
@@ -10,7 +10,10 @@
 # limitations under the License.
 import logging
 import re
-import subprocess as sub
+
+# A nosec comment is appended to the following line in order to disable the B404 check.
+# In this file the input of the module subprocess is trusted.
+import subprocess as sub  # nosec
```
Comment (awslitvit) [inline cli/src/pcluster/cli/commands/dcv_connect.py]: Same as above.

**Review by awslitvit — DISMISSED** [review]

File: cli/src/pcluster/api/util.py (hunk 1 above)
Comment (francesco-giordano) [reply cli/src/pcluster/api/util.py]: Done

File: cli/src/pcluster/cli/commands/dcv_connect.py (hunk 2 above)
Comment (francesco-giordano) [reply cli/src/pcluster/cli/commands/dcv_connect.py]: Done

**Review by awslitvit — APPROVED** [review]

**Pull request merged**